index.html

```diff
@@ -10,10 +10,8 @@
 <body>
     <div class="main-container">
         <header class="mobile-header">
-            <h1 id="name">Hans Paul Zizold</h1>
-            <button class="menu">
-                <img src="images/menu.png" alt="Menu">
-            </button>
+            <p id="name">Hans Paul Zizold</p>
+            <img src="images/menu.png" alt="Menu">
```

Comment:
[ OPTIONAL ]: It would be better to you use a `nav` tag for the menu button as its more professional 😃✔👍